wiki/Help_centre/Upgrading_to_lazer/en.md

```diff
@@ -0,0 +1,261 @@
+---
+stub: true
+tags:
+  - game client
+---
+
+# Upgrading to lazer
+
+osu!(lazer) is the next major update to the game. It is the culmination of several years of work behind the scenes to painstakingly reimplement the game.
+
+The eventual goal is that this version will be released as an update which will completely supersede the existing stable version of the game, but there's still some work to be done before that is feasible. At the end of the day, **the players** will dictate when this happens and we will continue to support the previous release until users have migrated across.
+
+"lazer" is a codename and will eventually be dropped as it becomes the primary release of the game. The rest of this document will refer to osu!(lazer) as "lazer" and osu!(stable) as "stable" for simplicity.
+
+## Feature comparison
+
+The following is a comprehensive list of the **current state** of lazer in comparison to stable. Note that this is a moving target — the end goal is to implement all the features that players care about over time.
+
+### Compatibility and performance
+
+| Feature | stable | lazer |
+| :-- | :-: | :-: |
+| Windows 8.0 and lower | ✅ | ❌ |
+| macOS / Linux | 🟠[^wine] | ✅ |
+| DirectX / Vulkan | 🟠[^compatibility-mode] | ❌[^coming-soon] |
+| Mobile support | ❌ | ✅ |
+| Multithreaded architecture | ❌ | ✅ |
+| Hardware-accelerated video | ❌ | ✅ |
+| UI scaling | ❌ | ✅ |
+| Custom rulesets (game modes) | ❌ | 🟠[^dll] |
+| De-duped file store | ❌ | ✅[^share-files] |
+| Tablet area adjustment | ❌ | ✅ |
+
+### UI and skinning
+
+| Feature | stable | lazer |
+| :-- | :-: | :-: |
+| Skin support | ✅ | 🟠[^gameplay-only] |
+| Song select grouping modes | ✅ | ❌ |
+| In-game skin / UI layout editing | ❌ | ✅ |
+
+### Gameplay and skinning
+
+| Feature | stable | lazer |
+| :-- | :-: | :-: |
+| Accurate performance point display | 🟠[^online] | ✅ |
+| Mod presets | ❌ | ✅ |
+| Per-mod settings | ❌ | ✅ |
+| New "fun" mods | ❌ | ✅ |
+| Combo colour normalisation[^normalisation] | ❌ | ✅ |
+| Hold for HUD | ❌ | ✅[^hold-for-hud] |
+| Offset calibration | ❌ | ✅[^offset-calibration] |
+| osu! sliders "snake" while dragging | ❌ | ✅[^can-disable] |
+| osu! player-friendly "note lock" | ❌ | ✅[^note-lock] |
+| osu!mania timing-based note colouring | ❌ | ✅ |
+
+### Online systems
+
+| Feature | stable | lazer |
+| :-- | :-: | :-: |
+| Score submission | ✅ | 🟠[^score-reset-balance] |
+| Beatmap leaderboards | ✅ | 🟠[^score-reset-isolated] |
+| Profile statistics | ✅ | ✅ |
+| Medals | ✅ | ❌ |
+| Performance points | ✅ | 🟠[^score-reset-isolated] |
+| Real-time chat | ❌ | ✅ |
+| Wiki / News / Changelog / Rankings | ❌ | ✅[^online-content] |
+| User profiles | ❌ | ✅ |
+| Beatmap listing | 🟠[^direct-supporter] | ✅ |
+| Changelog | ❌ | ✅ |
```

Comment:
```suggestion
```
Added back via the footnote patch by mistake?

Reply:
Weird, I explicitly removed this.